scripts/auto_build.sh

```diff
@@ -38,8 +38,9 @@ echo "Selected mode: $MODE"
 
 # Depending on the mode, execute different commands
 if [[ "$MODE" == "all" ]]; then
-    # Build all filters
-    yarn build
+    # Build all filters except ours to keep 1 hour update cycle of patches
+    # for our filters.
+    yarn build --skip=$ADGUARD_FILTERS
```

Comment:
now it looks more like `third-party` mode, not `all`

Reply:
will do it in AG-29172